paf-mvp-core-js/src/express/node.ts

```diff
@@ -149,8 +140,12 @@ export class Node implements INode {
           type: NodeErrorType.INVALID_QUERY_STRING,
           details: `Received Query string: '${data}' is not a valid Base64 string`,
         };
-        res.status(400);
-        res.json(error);
+        if (isRedirect) {
```

Comment:
(back from long holidays, probably missing context)
can you give the rationale why the `redirectWithError` is not done in the error handler anymore?

thanks

Reply:
Note this existing discussion: https://github.com/OneKey-Network/OneKey-implementation/pull/245#discussion_r956237314

We need to rework also the Error Handlers to handle errors properly (i.e not simply logging in the error handler and use `next()` more accurately in the handler).